catalog-rest-service/src/main/java/org/openmetadata/catalog/CatalogApplication.java

```diff
@@ -66,19 +70,25 @@
   @Override
   public void run(CatalogApplicationConfig catalogConfig, Environment environment)
       throws ClassNotFoundException, IllegalAccessException, InstantiationException, NoSuchMethodException,
-          InvocationTargetException {
+          InvocationTargetException, IOException {
 
     final JdbiFactory factory = new JdbiFactory();
     final Jdbi jdbi = factory.build(environment, catalogConfig.getDataSourceFactory(), "mysql3");
 
-    //    SqlLogger sqlLogger = new SqlLogger() {
-    //      @Override
-    //      public void logAfterExecution(StatementContext context) {
-    //        LOG.info("sql {}, parameters {}, timeTaken {} ms", context.getRenderedSql(),
-    //                context.getBinding().toString(), context.getElapsedTime(ChronoUnit.MILLIS));
-    //      }
-    //    };
-    //    jdbi.setSqlLogger(sqlLogger);
+    SqlLogger sqlLogger =
+        new SqlLogger() {
+          @Override
+          public void logAfterExecution(StatementContext context) {
+            LOG.debug(
+                "sql {}, parameters {}, timeTaken {} ms",
+                context.getRenderedSql(),
+                context.getBinding().toString(),
+                context.getElapsedTime(ChronoUnit.MILLIS));
+          }
+        };
+    if (LOG.isDebugEnabled()) {
```

Comment:
@sureshms Is it okay to enable SQL Logger if log level is set to debug?

Was contemplating removing commented out code, and figured we may need it for debugging